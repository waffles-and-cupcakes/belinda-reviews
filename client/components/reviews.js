import React from 'react';
import ReactDOM from 'react-dom';
import axios from 'axios';

class Reviews extends React.Component {
  constructor(props) {
    super(props);

    this.state = {
      total: 5,
      reviews: []
    }
  }

  componentDidMount() {
    axios.get('/rooms/1/reviews')
    .then((res) => {
      console.log('RESPONSE: ', res);
      this.setState({
        reviews: res.data
      });
    })
    .catch((err) => {
      console.log('error GET-ing /rooms/1/reviews')
    })
  }

  render() {
    return (
      <div> {JSON.stringify(this.state.reviews)}</div>
    );
  }
}

export default Reviews;