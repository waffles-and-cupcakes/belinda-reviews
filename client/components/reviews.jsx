import React from 'react';
import ReactDOM from 'react-dom';


class Reviews extends React.Component {

  render() {
    return (
      <section className="review-container"> 

        <div className="detail-container">
          <img className="user-photo" src="./pfp.jpg" />
          <div className="x-detail-container">
            <p> Kristen </p>
            <p> June 2017 </p>
          </div>
        </div>

        <div className="review-body"> 
          A beautiful space! We loved all the attention to 
          detail and local recommendations. Would love to se all
            their properties!
        </div>

      </section> 
    )
    
  }
}

export default Reviews;