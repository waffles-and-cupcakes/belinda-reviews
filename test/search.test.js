import React from 'react';
import {shallow} from 'enzyme';
import Search from '../client/components/search';
import renderer from 'react-test-renderer';
import reviews from '../client/mockData';


test('Search renders', () => {
  const component = renderer.create(
    <Search reviews={reviews} filterReviews={() => jest.fn()}/>
  )
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

// test('Search changes the styles when in focus', () => {
//   const component = renderer.create(
//     <Search reviews={reviews} filterReviews={() => (console.log('filtering reviews'))}/>
//   )
//   let tree = component.toJSON();
//   expect(tree).toMatchSnapshot();

//   // tree.props.handleChange();

//   // tree = component.toJSON();
//   // expect(tree).toMatchSnapshot();  

//   // tree.props.
// });
