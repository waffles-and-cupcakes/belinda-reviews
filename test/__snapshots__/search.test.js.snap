// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders 1`] = `
<div
  className="input-field col s12 m4"
>
  <input
    className="rev-search"
    onChange={[Function]}
    onKeyPress={[Function]}
    placeholder="Search reviews"
    value=""
  />
</div>
`;
